src/Views.php

```diff
@@ -435,4 +442,21 @@ protected function resolveVisitorId()
     {
         return $this->overriddenVisitor ?? $this->visitorCookieRepository->get();
     }
+
+    /**
+     * Resolve cache lifetime.
+     *
+     * @param  int|DateTime
+     * @return \Carbon\Carbon
+     */
+    protected function resolveCacheLifetime($lifetime): DateTime
+    {
+        if ($dateTime instanceof DateTime) {
```

Comment:
You may have already seen this but `$dateTime` doesn't exist here.

Reply:
I already fixed haha 😄. You probably saw an old commit: https://github.com/cyrildewit/eloquent-viewable/pull/146/commits/21ff267f06d18b7d38ea34b6ed652d0072fd9706. I still appreciate your feedback.